examples/schema/expected.txt

```diff
@@ -0,0 +1,26 @@
+nothing: null
+string: a string
+bool: false
+int: 0
+float: 0.1
+any: anything
+***
+nothing: a new string
+string: str
+bool: true
+int: 123
+float: 123.123
+any:
+- 1
+- 2
+- 4
+***
+nothing: a new string
+string: str
+bool: true
+int: 123
+float: 123.123
+any:
+- 1
+- 2
+- 4
```

Comment:
Similar to question above: in other example directories, we have `expected.txt` but I believe those are each referenced in a test in some way.

Reply:
same as above